ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/ShelleyHFC.hs

```diff
@@ -189,6 +201,11 @@ data ShelleyPartialLedgerConfig era = ShelleyPartialLedgerConfig {
       -- 'completeLedgerConfig' is called, as 'mkShelleyLedgerConfig' does
       -- some rather expensive computations that shouldn't be repeated too
       -- often (e.g., 'sgActiveSlotCoeff').
+      --
+      -- TODO Specialize this to a ShelleyLedgerConfig with unit instead of
```

Comment:
So do we need to convince Clarke of the necessity of input-output-hk/cardano-ledger-specs#2344 after all? My last comment there is saying it's merely a nice-to-have at this point. Is that still accurate?

Reply:
This is just a nice-to-have. I'll remove the comment for now.